src/NServiceBus.Autofac.AcceptanceTests/NServiceBus.Autofac.AcceptanceTests.csproj

```diff
@@ -1,18 +1,21 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net452;netcoreapp2.0</TargetFrameworks>
-    <DebugSymbols>True</DebugSymbols>
-    <NoWarn>PCR0001</NoWarn>
+    <TargetFrameworks>net462;netcoreapp2.0</TargetFrameworks>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
isnt `TreatWarningsAsErrors` already configured?
 https://github.com/Particular/NServiceBus.Autofac/blob/develop/src/Directory.Build.props#L4

Reply:
I didn't know it was in there. I will pull it out.